content/client-server-api/_index.md

```diff
@@ -1654,6 +1654,27 @@ An example of the capability API's response for this capability is:
 }
 ```
 
+### `m.get_login_token` capability
+
+This capability has a single flag, `enabled`, to denote whether the user
+is able to use [`POST /login/get_token`](/client-server-api/#post_matrixclientv1loginget_token)
+to generate single-use, time-limited tokens to log unauthenticated clients
+into their account.
+
+When not listed, clients SHOULD assume the user is unable to generate tokens.
```

Comment:
[MSC3882](https://github.com/matrix-org/matrix-spec-proposals/pull/3882) doesn't appear to specify how to behave in this case. The only implementations I found are in [matrix-js-sdk](https://github.com/search?q=repo%3Amatrix-org%2Fmatrix-js-sdk%20GET_LOGIN_TOKEN_CAPABILITY&type=code) and [matrix-react-sdk](https://github.com/search?q=repo%3Amatrix-org%2Fmatrix-react-sdk%20GET_LOGIN_TOKEN_CAPABILITY&type=code) and if I'm reading them correctly, they default to `enabled == false` if the capability is missing.

SHOULD seemed sufficient here because if a client would still call the endpoint, it would simply receive an error response which it must be prepared to handle in any case.